### DISCUSSION TITLE
Sync mobil devices

### DISCUSSION THREAD

**Discussion by PelleHanses** [docker-mailserver/docker-mailserver#3083] [Q&A] [answered]
Hi,
I am looking for a mail server that is easy to configure and quick to set up as we will be using it for several different organizations. Docker mail server seems to meet the simple requirements.
What I'm wondering is how do you sync email, contacts and calendar with mobile devices? Activesync does not appear to be implemented.

/Pelle

**Comment by casperklein** [comment, marked as answer]
There is no support for contacts/calendar in DMS. Only email is provided. If you use IMAP, all mails are stored on the mail server and can be accessed by different clients at the same time.
